shared/helpers.js

```diff
@@ -28,7 +28,6 @@ const printCurrentBrew = ()=>{
 		window.frames['BrewRenderer'].contentWindow.print();
 		//Force DOM reflow; Print dialog causes a repaint, and @media print CSS somehow makes out-of-view pages disappear
 		const node = window.frames['BrewRenderer'].contentDocument.getElementsByClassName('brewRenderer').item(0);
-		node.style.display='none';
```

Comment:
Any particular reason this was removed? This, plus the two lines below, were needed to prevent out-of-view pages from disappearing when the print dialog is opened. https://github.com/naturalcrit/homebrewery/pull/3491#issuecomment-2127685865

It's possible that bug no longer exists with newer Chrome versions, but still not sure what removing it here is needed for.

Reply:
ah I'm not sure, I think that is from one of @5e-Cleric earlier commits on the branch and I haven't noticed it.  Victor, do you know if that was needed?   I'll revert that and see what happens.

Reply:
I think i removed that because at the time the toolbar was the first child of the component and this line completely disabled it? don't recall, but we should be fine with removing it.

Reply:
reverted.  cls